### ISSUE TITLE
[FEATURE] Add widgets for EXT:sys_note

### ISSUE THREAD

**Issue by TYPO3IncTeam** [TYPO3-Documentation/Changelog-To-Doc#1024]
:information_source: View this commit [on Github](https://github.com/TYPO3/typo3/commit/aac4f7be152b043b1a85b333b6fa03e2a67eabc0)
:busts_in_silhouette: Authored by Philipp Kuhlmay typo3@treupo.de
:heavy_check_mark: Merged by Benni Mack benni@typo3.org

## Commit message

[FEATURE] Add widgets for EXT:sys_note

This adds some widgets for EXT:dashboard, which shows a list of the corresponding sys_notes inside the TYPO3 system.

The widgets are copied from EXT:treupo_sys_note_widgets which will not be developed further, as all the widgets are now inside the TYPO3 core.

Resolves: #104844
Releases: main
Change-Id: Iec0a1bef30919d96db282904885d8084ac02ac2c
Reviewed-on: https://review.typo3.org/c/Packages/TYPO3.CMS/+/85899
Tested-by: core-ci <typo3@b13.com>
Tested-by: Benni Mack <benni@typo3.org>
Reviewed-by: Frank Nägler <frank.naegler@typo3.com>
Reviewed-by: Benni Mack <benni@typo3.org>
Tested-by: Frank Nägler <frank.naegler@typo3.com>

## :heavy_plus_sign: Added files

<details>
<summary>13.3/Feature-104844-Dashboard-Sys_note-AddWidgetsForListingAllTheSys_notesInsideTheTYPO3System.rst</summary>

```rst
.. include:: /Includes.rst.txt

.. _feature-104844-1725617507:

====================================================================================
Feature: #104844 - Add widgets for listing all the sys_notes inside the TYPO3 system
====================================================================================

See :issue:`104844`

Description
===========

To make it easier for TYPO3 users to view all the internal notes (EXT:sys_note) in
their TYPO3 system, TYPO3 now offers dashboard widgets for each internal note type.
The backend user must have access to the sys_note table and view permission to the
page where the record is located.

Impact
======

TYPO3 users who have access to the :guilabel:`Dashboard` module and are
granted access to the new widgets can now add and use these widgets.

.. index:: Backend, ext:dashboard

```

</details>

**Comment by brotkrueml** [comment]
I suggest to add this to EXT:sys_note instead of EXT:dashboard as it belongs there.

**Comment by linawolf** [comment]
@brotkrueml good point, we didnt even have a label for that manual yet. Created one

**Comment by linawolf** [comment]
Will be solved here: https://forge.typo3.org/issues/106160

**Comment by linawolf** [comment]
Solved with https://review.typo3.org/c/Packages/TYPO3.CMS/+/88214
